web/src/utils/commons.ts

```diff
@@ -10,24 +10,29 @@
 //  distributed under the License is distributed on an "AS IS" BASIS,
 //  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //  See the License for the specific language governing permissions and
-//  limitations under the License. 
+//  limitations under the License.
 
 //Dashboard Manipulation Functions
 
 import dashboardService from "../services/dashboards";
 import { toRaw } from "vue";
 import { date } from "quasar";
 import moment from "moment";
+import { useStore } from "vuex";
```

Comment:
We should not be using `useStore` here. We can remove this, as function calls are async and it will not give you the store context when executed.